src/emails/templates/boilerplate/BoilerplateEmail.stories.tsx

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import type { Meta, StoryObj } from "@storybook/react";
+import { FC } from "react";
+import { SubscriberRow } from "knex/types/tables";
+import { Props, BoilerplateEmail } from "./BoilerplateEmail";
+import { StorybookEmailRenderer } from "../../StorybookEmailRenderer";
+
+const meta: Meta<FC<Props>> = {
+  title: "Emails/Email boilerplate",
+  component: (props: Props) => (
+    <StorybookEmailRenderer>
+      <BoilerplateEmail {...props} />
+    </StorybookEmailRenderer>
+  ),
+  args: {
+    subscriber: {
+      signup_language: "en",
+    } as SubscriberRow,
```

Comment:
Same as above re: `SubscriberRow`, I know we tend to pass this around but I think allow-listing the properties would be prevent someone from accidentally exposing something in the email that shouldn't be there.